### DISCUSSION TITLE
use_symbol_for_status in bash

### DISCUSSION THREAD

**Discussion by abmontiel** [starship/starship#1196] [General]
I'm having trouble getting the character option use_symbol_for_status to work in bash.

Could anyone help me out?

<img width="275" alt="Screen Shot 2020-05-13 at 5 24 10 PM" src="https://user-images.githubusercontent.com/56609934/81867485-c8f17600-953e-11ea-8120-15258e5e0d67.png">
<img width="131" alt="Screen Shot 2020-05-13 at 5 24 47 PM" src="https://user-images.githubusercontent.com/56609934/81867521-d3ac0b00-953e-11ea-8368-b6be479bc324.png">
<img width="294" alt="Screen Shot 2020-05-13 at 5 25 10 PM" src="https://user-images.githubusercontent.com/56609934/81867536-dad31900-953e-11ea-98b0-ddcccccccbbd.png">

**Comment by matchai** [comment]
That certainly seems strange. Please your bash configuration with us so we can help look into it. 
If you run `starship bug-report`, you can copy the populated template into this discussion.

**Comment by abmontiel** [comment]
#### Current Behavior
Character does not change when $? == 1

#### Expected Behavior
Expect character to change if 'false' is entered, as an example.

#### Additional context/Screenshots
In addition to my below .bashrc, here's my .bash_profile:

<img width="255" alt="Screen Shot 2020-05-14 at 9 38 46 AM" src="https://user-images.githubusercontent.com/56609934/81941739-3648e980-95c7-11ea-8af2-3b1986ea9fa7.png">


#### Environment
- Starship version: 0.41.0
- bash version: GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Copyright (C) 2007 Free Software Foundation, Inc.
- Operating system: Mac OS 10.14.6
- Terminal emulator: Apple_Terminal 421.2

#### Relevant Shell Configuration

```bash
alias ls='ls -Gra'
alias mv='mv -i'
alias rsync='rsync -hP'

eval "$(starship init bash)"

set -o vi
```

#### Starship Configuration

```toml
# ~/.config/starship.toml

# Replace the character with arrow
[character]
symbol = "➜"
error_symbol = "✗"
use_symbol_for_status = true


```

**Comment by matchai** [comment]
Can you try running the following:
```sh
 starship prompt --status="1"
```

If that works, please share the logs when you run the following:
```
RUST_LOG=starship starship prompt
```
